### PR TITLE
miatoll:Init CR 9.0

### PR DESCRIPTION
**Pull request by mohammadnt**

### CODE REVIEW

File: miatoll.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			"forum": "https://forum.xda-developers.com/t/beta-android-13-crdroid-9-0-official-miatoll.4430111/",
+			"gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/NikGapps-T/",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://orangefox.download/release/6332df463c05f43c193c1512",
+			"paypal": "",
+			"telegram": "https://t.me/+SmaG44X4RYp7iR7r",
+			"dt": "https://github.com/crdroidandroid/android_device_xiaomi_miatoll",
+			"common-dt": "https://github.com/crdroidandroid/android_device_xiaomi_sm6250-common",
```
Comment (gwolf2u) [inline miatoll.json]: No 13.0 branch

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: miatoll.json (hunk 1 above)
Comment (mohammadnt) [reply miatoll.json]: fixed

File: miatoll.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"forum": "https://forum.xda-developers.com/t/beta-android-13-crdroid-9-0-official-miatoll.4430111/",
+			"gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/NikGapps-T/",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://orangefox.download/release/6332df463c05f43c193c1512",
+			"paypal": "",
+			"telegram": "https://t.me/+SmaG44X4RYp7iR7r",
+			"dt": "https://github.com/crdroidandroid/android_device_xiaomi_miatoll",
+			"common-dt": "",
```
Comment (gwolf2u) [inline miatoll.json:24]: no common dt now?

Comment (mohammadnt) [reply miatoll.json:24]: No it does not have common dt

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**